### PR TITLE
METRON-1445: Update performance tuning guide with more explicit parameter instructions

### PR DESCRIPTION
**Pull request by mmiklavc**
## Contributor Comments

https://issues.apache.org/jira/browse/METRON-1445

The current performance tuning guide can be challenging to follow for those not familiar with the system. This PR updates the guide to provide details about how and where to find tuning parameters along with details as to how the properties are materialized in a running topology.

Additionally, a supplement to the tuning guide has been provided to pull Metron cluster debugging info into a tarball. As pertains to the perf guide, it captures the state of Ambari configuration, flux files, properties files, and zk config. This helps users confirm system state as they make tuning changes.

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
n/a

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered by building and verifying the site-book? If not then run the following commands and the verify changes via `site-book/target/site/index.html`:

  ```
  cd site-book
  mvn site
  ```

### CODE REVIEW

**Comment by cestella** [comment]
+1 by inspection, this is great detail.

File: metron-platform/metron-common/README.md [hunk 1]
```diff
@@ -400,3 +401,34 @@ __Side Effects__
...
 The class is not thread-safe and makes no attempt at keeping multiple threads from modifying the same markers.
 
+# Metron Debugging
+
+A Python script is provided for gathering information useful in debugging your Metron cluster. Run from the node that has Metron installed on it. All options listed below are required.
+
+_Note:_ Be aware that no anonymization/scrubbing is performed on the captured configuration details.
+
+```
+# ./cluster_info.py -h
```
[the remark this answers is not in the document]
Comment (mmiklavc) [reply metron-platform/metron-common/README.md]: @nickwallen fixed to reference $METRON_HOME/bin

**Comment by nickwallen** [comment]
+1 Thanks

File: metron-platform/Performance-tuning-guide.md [hunk 2]
```diff
@@ -111,15 +138,146 @@ __spout-config.json__
...
+|                             | n/a                                        | n/a                                                    | setOffsetCommitPeriodMs                                | line 108, id: kafkaConfig           | n/a                             | Kafka consumer client property         |
+| Output bolt                 | hdfs_writer_parallelism                    | hdfs.writer.parallelism                                | parallelism                                            | line 133, id: hdfsIndexingBolt      | n/a                             |                                        |
+|                             | n/a                                        | n/a                                                    | hdfsSyncPolicy <see notes below>                       | line 47, id: hdfsWriter             | n/a                             | See notes below about adding this prop |
```
Comment (justinleet) [inline metron-platform/Performance-tuning-guide.md]: ```<see notes below>``` to ```&lt;see notes below&gt;```

Unfortunately, it seems like the newer Doxia version parses slightly differently than before, but on the plus side it finishes.

Comment (mmiklavc) [reply metron-platform/Performance-tuning-guide.md]: Thanks @justinleet - fixed and checked locally. Will wait for Travis to run and merge this.

**Pull request closed without merging**